_tuning-your-cluster/availability-and-recovery/segment-replication/index.md

```diff
@@ -51,12 +56,12 @@ PUT /my-index1
 In segment replication, the primary shard is usually generating more network traffic than the replicas because it copies segment files to the replicas. Thus, it's beneficial to distribute primary shards equally between the nodes. To ensure balanced primary shard distribution, set the dynamic `cluster.routing.allocation.balance.prefer_primary` setting to `true`. For more information, see [Cluster settings]({{site.url}}{{site.baseurl}}/api-reference/cluster-api/cluster-settings/).
 
 Segment replication currently does not support the `wait_for` value in the `refresh` query parameter.
```

Comment:
Do we need to keep all limitations at one place under limitation sections ?

Reply:
Yes, good catch. Change made.

Reply:
Do we have documentation for the refresh parameter?

Reply:
Revised